decorators/overridable.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withPropsFromContext } from 'react-context-props'
+
+const overridable = (styles = {}, designName) => (Target) => {
+  const OverridableComponent = React.createClass({
```

Comment:
Can we avoid using `createClass` here? I think it's not compatible w/ Preact and it may be deprecd soon.